CHANGELOG.md

```diff
@@ -48,6 +48,9 @@ release.
 - Clarify that `http/dup` has higher precedence than `http` in case both values are present
   in `OTEL_SEMCONV_STABILITY_OPT_IN`
   ([#249](https://github.com/open-telemetry/semantic-conventions/pull/249))
+- Rename metrics `jvm.buffer.usage` to `jvm.buffer.memory.usage`
```

Comment:
This is a "breaking" change right? Maybe indicate here in the changelog?

Reply:
```suggestion
- BREAKING: Rename metrics `jvm.buffer.usage` to `jvm.buffer.memory.usage`
```